### ISSUE TITLE
False positive results in tryFindingTappableViewWithAccessibilityLabel

### ISSUE THREAD

**Issue by ivan-golub** [kif-framework/KIF#514]
Im using 
[tester tryFindingTappableViewWithAccessibilityLabel:@"Back" traits:UIAccessibilityTraitButton error:nil]
And it returns YES, but there are no back button on the screen.
Also [tester waitForTappableViewWithAccessibilityLabel:@"Back" traits:UIAccessibilityTraitButton]; 
and [tester tapViewWithAccessibilityLabel:@"Back" traits:UIAccessibilityTraitButton]; are not not failing

This back button is system navigation button, which is not shown because Im on root controller. Here is a description of UIView returned from [self waitForTappableViewWithAccessibilityLabel:@"Back" traits:UIAccessibilityTraitButton];
<_UINavigationBarBackIndicatorView: 0x1743eec00; frame = (8 11.5; 13 21); alpha = 0; opaque = NO; userInteractionEnabled = NO; layer = <CALayer: 0x176e37240>> - Back

You cannot tap view with alpha 0, view become tappable only at alpha = 0.02-0.03 so basically it is hidden

Reproduced on iPhone 6 ios 8.1, KIF 3.0.8, xCode 6.1.1
